libcxxabi/src/CMakeLists.txt

```diff
@@ -16,9 +16,14 @@ set(LIBCXXABI_SOURCES
   # Internal files
   abort_message.cpp
   fallback_malloc.cpp
-  private_typeinfo.cpp
 )
 
+# private_typeinfo.cpp needs to be built by enabling rtti.
```

Comment:
Do you mean ”private_typeinfo.cpp needs to be built with RTTI enabled”?

Reply:
Thanks, fixed in the commit https://github.com/SNSystems/llvm-project-prepo/pull/145/commits/7ef52d812cc1e524350d3241e5bdf90077da4b3e.